float32[] commanded_angles
float32 execution_time
# float32 aa
